.gitignore

```diff
@@ -22,6 +22,6 @@ yarn-error.log*
 
 #TODO add .env to your gitignore
 __snapshots__
-
+sample.env
```

Comment:
As mentioned in PR #45 

We don't want to ignore this but rather we want this to be a template of what `.env` should look like.

Reply:
noted with thanks